src/evm.ts

```diff
@@ -40,3 +41,17 @@ export const decodeRevertString = (returndata: string) => {
 
   return goDecode.data;
 };
+
+export const estimateMulticallGasLimit = async (
+  contract: SelfMulticall,
+  calldatas: string[],
+  fallbackGasLimit: ethers.BigNumber | undefined
+) => {
+  const goEstimateGas = await go(async () => contract.estimateGas.multicall(calldatas));
+  if (goEstimateGas.success) {
+    // Adding a extra 10% because multicall consumes less gas than tryMulticall
```

Comment:
nit (I see that the same nit suggestion could be done for Airseeker v2): 
```suggestion
    // Adding an extra 10% because multicall consumes less gas than tryMulticall
```